src/Command/PluginAssetsTrait.php

```diff
@@ -14,70 +14,32 @@
  * @since         3.0.0
  * @license       https://opensource.org/licenses/mit-license.php MIT License
  */
-namespace Cake\Shell\Task;
+namespace Cake\Command;
 
-use Cake\Console\ConsoleOptionParser;
-use Cake\Console\Shell;
 use Cake\Core\Configure;
 use Cake\Core\Plugin;
 use Cake\Filesystem\Filesystem;
 use Cake\Utility\Inflector;
 
 /**
- * Task for symlinking / copying plugin assets to app's webroot.
+ * trait for symlinking / copying plugin assets to app's webroot.
+ * @internal
  */
-class AssetsTask extends Shell
+trait PluginAssetsTrait
```

Comment:
Instead of a trait you could make simple class that exposes the required methods as public methods. We could do that change in a follow-up pull request as well though.

Reply:
probably yes but I no have idea for good class/methods name and place it 
I marked this as `@internal`
I think we can do it in next PR